tasks/aws/src/main/java/com/walmartlabs/concord/plugins/aws/EcrTask.java

```diff
@@ -63,29 +66,38 @@ public TaskResult execute(Variables input) {
     private TaskResult describeImages(Variables input) {
         var region = assertRegion(input, "region");
         var repositoryName = input.assertString("repositoryName");
+        var maxResults = input.getInt("maxResults", 100);
         var verbose = input.getBoolean("verbose", false);
```

Comment:
maybe `debug` instead of `verbse`? It seems that in all other tasks this parameter is called debug:)

and default value is `context.processConfiguration().debug()`